packages/clay-css/src/scss/atlas/variables/_globals.scss

```diff
@@ -12,60 +12,32 @@ $c-unset: $clay-unset !default;
 
 $clay-unset-placeholder: clay-unset-placeholder !default;
 
+// Settings
+
```

Comment:
Hey @pat270 sounds good to me, but a question, all these variables we added are coming from globals, right? I have the same concern you raised in the issue about having to keep two files and this can get confusing as we have to iterate and is subject to more failure.

I understand that the problem here is to lower errors like `Undefined variable` because the developer might be trying to extend from `atlas`, to avoid that then he would have to add the globals together right? Why not have an atlas "endpoint" that joins the files needed for an atlas theme to work or just join globals and the atlas variables? wouldn't that make more sense?

Reply:
We do that with `atlas.scss` and `_atlas-variables.scss`. This pr isn't necessary. It's a quality of life adjustment for devs that want to extend Atlas. In Liferay Themes, there is only one "safe" file for overwriting variables, `_clay_variables.scss`. It's the first variable import in `atlas.scss`. 

```
@import '_license-text';

@import 'functions/_global-functions';

@import '_clay_variables.scss'; // <---

@import 'atlas/_variables';

@import '_variables';

@import '_mixins';

@import '_components';
```

It gets annoying when you copy paste a component map you want to modify, but errors due to a global variable missing at the top of `_clay_variables.scss` (e.g., I only copied over the global variables defined in Atlas). Currently Atlas only changes the global variables that need to be changed. By defining all variables available in both themes, it makes it a lot easier to copy paste and go.

Reply:
I see. I understand this is to improve the development experience and not anything against it but can we do something that we can solve without needing to duplicate unused variables? I imagine that if in the global variables of the atlas we already import the global variables, that could resolve it?

This is also a case where one just wants to import the `@import 'atlas/_variables';` right?

Reply:
Can't think of another way other than documenting copy paste https://github.com/liferay/clay/blob/master/packages/clay-css/src/scss/atlas/variables/_globals.scss and https://github.com/liferay/clay/blob/master/packages/clay-css/src/scss/variables/_globals.scss at the top of your `_clay_variables.scss` file. It would result in duplicates for them.

Inserting `_clay_variables.scss` later in the import chain will cause the `!default` pattern not to work if they want to make their theme extendable or if they created a variable theme based on it due to Clay using it.

Reply:
I thought Sass could avoid import duplicates but in fact, it shouldn't anyway... This makes sense @pat270, maybe to avoid all this we would have to modify the import order and locations. Well, I'm fine with moving on with that.